src/styles/cardStyles.scss

```
@@ -0,0 +1,108 @@
+.text-Main {
+  transition-duration: $duration;
+  color: $Main-accent;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 0 0;
```

reset default margin for `p` tag and `h3` to not reset this value each time
```suggestion
  margin-top: 40px;
```